Poker/Deck.cs

```diff
@@ -7,13 +7,24 @@ namespace Poker
 {
     public class Deck
     {
-        public List<Card> listOfCards { get; }
+        public List<Card> ListOfCards { get; }
```

Comment:
All fields in a class should start with capitals.

If you are creating a variable in a method that you will get don't need when the method finishes, then that starts with a lower case